packages/core/src/external/surescripts/fhir/shared.ts

```diff
@@ -69,6 +71,24 @@ export function deduplicateByCoding<R extends Medication | Condition>(
   return masterResource;
 }
 
+export function deduplicateByKey<R extends Resource, K extends keyof R>(
```

Comment:
Part of the motivation for producing a deduplicated conversion bundle is that any broken references can be triaged to dangerouslyDeduplicateFhir (as I learned with MedicationDispense).